### PR TITLE
updated readme and contributing for putt-putt-dataset

### PR DESCRIPTION
**Pull request by SarahMalykke**
Updated the readme and contributing for putt-putt @jessb0t 🙂

### CODE REVIEW

File: materials/flanker_golf_tasks/readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Task1
+# Flanker and golf tasks for putt-putt-dataset
 
 ### Instructions
-This folder contains the protocols, scripts, and stimuli used to create this experimental task. Replace "Task1" in the header above with a descriptive name for the task; rename the "task1" folder in like manner.
+This folder contains the protocols, scripts, and stimuli used to create this experimental task.
```
Comment (jessb0t) [inline materials/flanker_golf_tasks/readme.md:4]: But this folder doesn't actually contain anything yet?  It should have the full PsychoPy experiment for the flanker (under scripts) and then maybe a detailed overview of the setup of the golf-putting task?  Feel free to collapse the levels and make the flow more intuitive within the "materials" folder as a whole.  The purpose of this folder is to enable replicability of your methods.

Comment (jessb0t) [reply materials/flanker_golf_tasks/readme.md:4]: Also, please separate the flanker and golf task materials, since they are independent. :)

**Review by jessb0t — CHANGES_REQUESTED** [review]
@SarahMalykke Thank you for your work on putting together the public face and traceability of the putt-putt-dataset!

I made several comments on specific lines of code.  Also, please update the questionnaires section.  You can look to either [rwe](https://github.com/NDCLab/rwe-dataset/tree/main/materials/questionnaires) or [readAloud-valence](https://github.com/NDCLab/readAloud-valence-dataset/tree/dev/materials/questionnaires) for suggestions on ways to tackle this!

Thank you!

**Comment by SarahMalykke** [comment]
@jessb0t I updated the readme file 🙂 I'll work on the materials folder next time I'm in AHC5, so I can get the flanker files and take pictures of my golf setup

**Review by jessb0t — APPROVED** [review]

**Pull request merged**